pennylane_forest/qpu.py

```diff
@@ -190,7 +163,7 @@ def __init__(
         self.calibrate_readout = calibrate_readout
         self.wiring = {i: q for i, q in enumerate(self.qc.qubits())}
 
-    def expval(self, observable):
+    def expval(self, observable, shot_range, bin_size):
```

Comment:
type hints, at least for the new args?

Reply:
These aren't new args we've added, we are just updating the signature to match the [PennyLane `QubitDevice`](https://docs.pennylane.ai/en/stable/code/api/pennylane.QubitDevice.html#pennylane.QubitDevice.expval) parent class this ultimately derives from. Their classes don't have type hints, so I don't know if we'd want to extrapolate those for them here. 

I don't think it would improve ergonomics much anyways since this function wouldn't usually be called directly by users. Rather, they would setup a `QNode` and let PennyLanes `expval` do the rest of the work ([example](https://docs.pennylane.ai/en/stable/code/api/pennylane.expval.html)).

Reply:
I will add default arguments for `shot_range` and `bin_size` though, since those are optional in `QubitDevice`